budproj/business
# Adding notification and email for mention to create kr checkin event

## 🎢 Motivation

Create a notification and email trigger into kr check in create event for new mark user feature.

## 🔧 Solution

Into event, create a dispatch to notification and email.

## 🚨  Testing

1. Create a checkin
2. On comment mark a user
3. Check if for notification and email 

## 🃏 Task Card

Link to issue on Jira

- [`BUD22-123`](https://)

## 🔗 Related PRs

This PR is related to some other PRs in different services, they are:

- [`project#PR_NUMBER`](https://)

## 🍩 Validation

Who tested this PR?

### Canary


### Dev
